docs/pages/extensibility/CustomServerMiddleware.md

```diff
@@ -85,23 +85,33 @@ middleware:
 A custom middleware implementation needs to return a function with the following signature:
 ````javascript
 /**
- * Custom UI5 Server middleware example
+ * Custom UI5 Server middleware API
  *
  * @param {object} parameters Parameters
- * @param {object} parameters.resources Resource collections
- * @param {module:@ui5/fs.AbstractReader} parameters.resources.all Reader or Collection to read resources of the
- *                                        root project and its dependencies
- * @param {module:@ui5/fs.AbstractReader} parameters.resources.rootProject Reader or Collection to read resources of
- *                                        the project the server is started in
- * @param {module:@ui5/fs.AbstractReader} parameters.resources.dependencies Reader or Collection to read resources of
- *                                        the projects dependencies
- * @param {object} parameters.middlewareUtil Specification version dependent interface to a
- *                                        [MiddlewareUtil]{@link module:@ui5/server.middleware.MiddlewareUtil} instance
+ * @param {object} parameters.resources Readers for accessing resources
+ * @param {module:@ui5/fs.AbstractReader} parameters.resources.all
+ *      Reader to access resources of the root project and its dependencies
+ * @param {module:@ui5/fs.AbstractReader} parameters.resources.rootProject
+ *      Reader to access resources of the root project
+ * @param {module:@ui5/fs.AbstractReader} parameters.resources.dependencies
+ *      Reader to access resources of the project's dependencies
+ * @param {@ui5/server.middleware.MiddlewareUtil} parameters.middlewareUtil
+ *      Specification version-dependent interface to a
+ *      MiddlewareUtil instance. See the corresponding API reference for details.
```

Comment:
What API reference? It's rather the corresponding documentation page, right?

Reply:
There isn't much documentation: https://sap.github.io/ui5-tooling/pages/extensibility/CustomServerMiddleware/#helper-class-middlewareutil

The API reference is more helpful imho: https://sap.github.io/ui5-tooling/api/module-@ui5_server.middleware.MiddlewareUtil.html

Reply:
@RandomByte would you like to include the mentioned url as a link in the comment?

Reply:
Done